docs/installation.md

```diff
@@ -53,7 +53,7 @@ Information about the binaries in the release tarball:
 
 |           Binary Name          |      Support      |   OS  | Architecture |
 |:------------------------------:|:-----------------:|:-----:|:------------:|
-|          cri-containerd        | seccomp, apparmor | linux |     amd64    |
+|            containerd          | seccomp, apparmor | linux |     amd64    |
```

Comment:
Remove `cri-containerd` from the release tarball above.

There are several places not updated yet in this document.

Reply:
I did remove cri-containerd from the release tarball..?

Reply:
Look at the paragraph above.

Reply:
done'